code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4362,6 +4362,35 @@
 	reagents.add_reagent(TENDIES, 3)
 	bitesize = 2
 
+//////////////////VOX CHICKEN//////////////////
+
+/obj/item/weapon/reagent_containers/food/snacks/vox_nuggets
+	name = "Vox Nuggets"
+	desc = "Looks awful and off-colour, you wish you'd gone to Cluckin' Bell instead."
+	icon_state = "vox_nuggets"
+	item_state = "kfc_bucket"
+	trash = /obj/item/trash/chicken_bucket
+	food_flags = FOOD_MEAT
+	filling_color = "#4A75F4"
+	base_crumb_chance = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/vox_nuggets/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 6)
+	bitesize = 1
```

Comment:
these have no reason to be in New, move it above in the type definition

Reply:
That's how a lot of foods do it, doesn't seem to matter, I did change it however.